internal/view/content.html

```diff
@@ -48,15 +50,26 @@
 			el: '#content-scene',
 			mixins: [basePage],
 			data: {
-				modified: "$$.Book.Modified$$"
+				created: "$$.Book.Created$$"
 			},
 			methods: {
-				localtime() {
-					var strTime = this.modified.replace(" ", "T");
-					if (!strTime.endsWith("Z")) {
-						strTime += "Z";
+				createdtime() {
+					var strCreatedTime = "$$.Book.Created$$".replace(" ", "T");
+                    var strModifiedTime = "$$.Book.Modified$$".replace(" ", "T");
+					if (!strCreatedTime.endsWith("Z")) {
+						strCreatedTime += "Z";
 					}
-					return dayjs(strTime).format("D MMMM YYYY, HH:mm:ss");
+					if (!strModifiedTime.endsWith("Z")) {
+						strModifiedTime += "Z";
+					}
+                    return `${dayjs(strCreatedTime).format("D MMMM YYYY")}`;
+				},
+				modifidetime() {
```

Comment:
Can we use `camelCase` here? (also there's a typo)

```suggestion
				modifiedTime() {
```